ext/vscode/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 0.8.0-alpha.1 (Unreleased)
+## 0.8.0 (2023-11-15)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- [[#2541]](https://github.com/Azure/azure-dev/pull/2541) Support has been added for the Azure Developer CLI to fetch authentication tokens from VSCode, reducing the need to re-authenticate.
+- [[#2771]](https://github.com/Azure/azure-dev/pull/2771) Commands to enable or disable Dev Center mode have been added. Dev Center mode lorem ipsum dolor sit.
```

Comment:
@savannahostrowski what would you like me to say about Dev Center mode here?

Reply:
Dev Center mode allows you to use Azure Deployment Environments for environment definitions (which include infrastructure as code and other manifests).

@wbreza might have a more eloquent way of phrasing this.

Reply:
Dev center mode allows `azd` to leverage Infrastructure as Code (IaC) templates from Dev Centers centrally managed catalogs, manage remote Azure Deployment Environments (ADE) and seamlessly deploy applications to ADE environments using existing `azd deploy` commands.